modules/openapi-generator/src/main/resources/typescript-nestjs/package.mustache

```diff
@@ -43,7 +43,7 @@
     "@nestjs/testing": "~{{nestVersion}}",
     "@types/express": "^4.16.0",
     "@types/jest": "^24.0.15",
-    "@types/node": "^14.8.2",
```

Comment:
These types are so old that some definitions clash with the current typescript version.